src/main/java/org/sagebionetworks/web/client/presenter/SynapseForumPresenter.java

```diff
@@ -34,50 +27,40 @@
 	SynapseForumView view;
 	//use this token to navigate between threads within the discussion tab
 	ParameterizedToken params;
-	 
-	NewDiscussionThreadModal newThreadModal;
-	DiscussionThreadListWidget threadListWidget;
+
 	SynapseAlert synAlert;
-	DiscussionForumClientAsync discussionForumClient;
 	SynapseClientAsync synapseClient;
-	AuthenticationController authController;
 	GlobalApplicationState globalApplicationState;
-	private String forumId;
 	
 	MarkdownWidget wikiPage;
 	WikiPageKey pageKey;
 	Boolean isCurrentUserModerator;
 	CookieProvider cookies;
-	
+	ForumWidget forumWidget;
+
+	private String areaToken = null;
```

Comment:
areaToken is never set.  Please expose a method in ParameterizedPlace to get the ParameterizedToken (instead of recreating based on the place.toToken())